hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/CacheKey.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.data;
+
+import java.io.Serializable;
+
+public class CacheKey implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private String tenantId;
+    private String dataIdPrefix;
+    private String dataIdSuffix;
```

Comment:
What is the logic for storing suffix on the dataId ?
Is this CacheKey related to the publishing workflow ?